### PR TITLE
feat(launchpad): upgrade nodes version

### PR DESCRIPTION
**Pull request by mazzi**
### Description

New feature, update node versions from launchpad.

Screenshot 2024-11-05 at 17 34 25

Screenshot 2024-11-05 at 17 34 10

### CODE REVIEW

File: node-launchpad/src/components/status.rs [hunk 1]
```diff
@@ -591,6 +632,40 @@ impl Component for Status<'_> {
...
+                let upgrade_nodes_args = UpgradeNodesArgs {
+                    action_sender,
+                    connection_timeout_s: 5,
+                    do_not_start: true,
+                    custom_bin_path: None,
+                    force: false,
+                    fixed_interval: None,
+                    peer_ids,
+                    provided_env_variables: None,
+                    service_names,
```
Comment (RolandSherwin) [inline node-launchpad/src/components/status.rs:663]: We can use a empty `vec![]` here and we'd upgrade all the nodes. We don't have to pass in the service names.

**Review by jacderida — APPROVED** [review]

**Review by RolandSherwin — APPROVED** [review]
LGTM! 🚀

File: node-launchpad/src/components/status.rs (hunk 1 above)
Comment (RolandSherwin) [reply node-launchpad/src/components/status.rs:663]: No harm with the current impl, so resolving this.

Comment (mazzi) [reply node-launchpad/src/components/status.rs:663]: Follow up PR :)

**Pull request merged**